posthog/tasks/usage_report.py

```diff
@@ -816,7 +825,6 @@ def get_teams_with_recording_bytes_in_period(
 
 @shared_task(**USAGE_REPORT_TASK_KWARGS, max_retries=0)
 def capture_report(
-    self: celery.Task,
```

Comment:
Why remove this one?

Reply:
`*` catches everything and it wasn't used. reverted